### PR TITLE
Improve server packaging

### PR DESCRIPTION
**Pull request by gcotelli**
Moved methods used exclusively in the server code to JRPC-Server package, in order to not have this methods if you load only the Client group.

### CODE REVIEW

File: src/JRPC-Server/JRPCHTTPServer.class.st [hunk 1]
```diff
@@ -36,11 +36,6 @@ JRPCHTTPServer >> initialize [
 				to: [ :request :response | response entity: ( ZnEntity json: ( self handleJSON: request contents ) ) ] )
 ]
 
-{ #category : #accessing }
-JRPCHTTPServer >> port [
```
Comment (juliendelplanque) [inline src/JRPC-Server/JRPCHTTPServer.class.st]: This method is yet never sent but maybe it should be kept no?

**Review by juliendelplanque — APPROVED** [review]
Looks nice.

Thanks for your changes.

Just need to discuss `#port` method removal.

File: src/JRPC-Server/JRPCHTTPServer.class.st (hunk 1 above)
Comment (gcotelli) [reply src/JRPC-Server/JRPCHTTPServer.class.st]: If you think is useful we can keep it. Anyway there are other server options that can be exposed, like `#debugMode` and `#serverUrl`, probably I will need it in the future

**Comment by gcotelli** [comment]
I've recovered the `port` method and added `debugMode` option because it's better for debugging in case of test failures.
I've also added some tests to cover the notification API

**Comment by juliendelplanque** [comment]
Ok nice, waiting for CI to build the PR and I merge and create a new minor release.

**Comment by gcotelli** [comment]
Finally travis finished running the builds. It's ready to merge now :)

**Comment by juliendelplanque** [comment]
New release after integration of this PR. https://github.com/juliendelplanque/JRPC/releases/tag/v2.1.0

**Pull request merged**